.travis.yml

```diff
@@ -13,10 +13,23 @@ before_install:
   - carthage bootstrap --use-submodules --no-build
 
 script:
+  - carthage build --no-skip-current
+  - carthage archive APIKit
```

Comment:
Thus, we should conditionally run those with checking `[ "${TRAVIS_PULL_REQUEST}" = "false" ]`